### DISCUSSION TITLE
Getting RSSI from different Crazyradio dongle at the same time

### DISCUSSION THREAD

**Discussion by scf-ku** [bitcraze/discussions#1578] [Q&A]
Hi,

I have succeed to get RSSI between Crazyflie and Crazyradio dongle with this issue(https://github.com/orgs/bitcraze/discussions/1548).

And now I'm trying to get RSSI from two different Crazyradio dongle at the same time with single crazyflie.

Are there any effective methods to address the above issue?
(Is it possible by changing Crazyradio dongle's address or something?)

Best Regards.

**Comment by gemenerik** [comment]
Note that the RSSI you are receiving here is on the CF side.
We do not support two simultaneous connections (so no 2 radios 1 cf), but I guess you can indeed command a change in channel like in the `examples/radio/radio-test.py` to a channel your second radio is listening to.
